types.d.ts

```diff
@@ -0,0 +1,5009 @@
+/* tslint:disable */
```

Comment:
Consider how this can be autogenerated and included in `dist` to avoid including in source control